plugin/rename.py

```diff
@@ -82,7 +82,10 @@ def run(
         point: Optional[int] = None
     ) -> None:
         if position is None:
-            pos = get_position(self.view, event, point)
+            tmp_pos = get_position(self.view, event, point)
+            if tmp_pos is None:
+                return
+            pos = tmp_pos
```

Comment:
Too many positions :D 

<img width="1076" alt="Screenshot 2021-04-22 at 23 10 09" src="https://user-images.githubusercontent.com/22029477/115785621-fe921680-a3bf-11eb-8ec0-9eb84aefa458.png">

Why not just:
```
pos = get_position(self.view, event, point)
if pos is None:
    return
if new_name:
```

Reply:
Sorry for nitpicking

Reply:
pyright will report type issue in lambda below.
And it's correct. Due to how python works, the variable can change before the lambda runs.